src/west/commands/project.py

```diff
@@ -74,6 +74,7 @@ def do_run(self, args, user_args):
             shutil.move(args.cache, project.abspath)
 
             _update_key(config, 'manifest', 'path', project.path)
+        _update_key(config, 'zephyr', 'base', project.path)
```

Comment:
This should be updated to ensure the project refering to a `zephyr` project is used instead.

Reply:
Indeed; this needs to loop over the projects to find the project with path `zephyr`.

Also, shouldn't this be an abspath?

Reply:
as long as this is inside `west` managed project, this does not matter.
But if user add `zephyr.base` to ~/.westconfig for out of tree (not supported yet), then he would need to specify abs path.

Reply:
but abs path would still be correct as well, so if abs path is preferred, i will change.

Reply:
but then again.
`manifest.path`  is not an abs path.

Reply:
now looping.
Actually, it feels kind of silly to loop in order to find the project that has `path == zephyr` cause the resulting setting in that case will actually become `zephyr.base == zephyr` !!!

On the other  hand, it might be valuable in future if deciding to have other way of specifying the `zephyr` project besides the `path`

Reply:
I took a second look at the code, and instead of requesting abspath here, then I will ensure abspath later.